keptn/lifecycle-toolkit
# feat: use helmify to release our helm chart

## This Pr

- adds an autogenerated helm chart
-  fix related pipelines
-  updates chart README

## How to test

`make helm-package`
check helm/chart
`helm template /helm/chart`
`helm upgrade --install klt --namespace=klt ./helm/chart  --debug --set fullnameOverride=klt --create-namespace`


https://github.com/keptn/lifecycle-toolkit/actions/runs/4260961532/jobs/7414795889
